e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -219,16 +219,16 @@ private void verifyJsonFromfileDownload(String fileContent, JSONArray extension)
         Assert.assertTrue(validFields(fileJson));
         Assert.assertEquals("ExplanationOfBenefit", fileJson.getString("resourceType"));
         Assert.assertEquals(0, fileJson.getInt("precedence"));
-        String carrierString = fileJson.getString("id");
-        Assert.assertTrue(carrierString.startsWith("carrier"));
+        String idString = fileJson.getString("id");
+        Assert.assertTrue(idString.startsWith("carrier") || idString.startsWith("outpatient"));
```

Comment:
Should we use this opportunity to cover all the bases? The valid values include: carrier, dme, hha, hospice, inpatient, outpatient, snf.